tests/kafkatest/tests/core/transactions_test.py

```diff
@@ -246,8 +246,9 @@ def setup_topics(self):
     @matrix(failure_mode=["hard_bounce", "clean_bounce"],
             bounce_target=["brokers", "clients"],
             check_order=[True, False],
-            use_group_metadata=[True, False])
-    def test_transactions(self, failure_mode, bounce_target, check_order, use_group_metadata, metadata_quorum=quorum.all):
+            use_group_metadata=[True, False],
+            use_new_coordinator=[True, False])
```

Comment:
I suppose that this one fails because we haven't implemented the transactional offsets commits yet. I also wonder if they are really executed with KRaft.

Reply:
hmm weirdly it passes 😕 I see quorum.all specified in the test args but I think it should be in the matrix so all the combinations run. I'll change it and run it to confirm. Thanks for the catch!

Reply:
https://jenkins.confluent.io/job/system-test-kafka-branch-builder/5970/

Reply:
All the kraft tests failed in that one...